app/src/organisms/Desktop/Devices/RobotSettings/RobotSettingsCamera/hooks/useCameraUsageSettings.ts

```diff
@@ -0,0 +1,99 @@
+import { useEffect, useState } from 'react'
+
+import { useCamera, useUpdateCamera } from '@opentrons/react-api-client'
+
+const CAMERA_POLLING_INTERVAL_MS = 5000
+
+export interface UseCameraUsageSettingsResult {
+  /* Whether the camera is generally enabled. No other settings
+   * are configurable if this setting is false. */
+  isCameraEnabled: boolean
+  isLiveVideoEnabled: boolean
+  isRecoveryCaptureEnabled: boolean
+  toggleCameraEnabled: () => void
+  toggleLiveVideoEnabled: () => void
+  toggleRecoveryCaptureEnabled: () => void
+}
+
+// general camera usage settings.
+export function useCameraUsageSettings(): UseCameraUsageSettingsResult {
+  const { data: cameraData } = useCamera({
+    refetchInterval: CAMERA_POLLING_INTERVAL_MS,
+  })
+  const { mutateAsync: updateCamera } = useUpdateCamera()
+
+  const [isCameraEnabled, setIsCameraEnabled] = useState(true)
+  const [isLiveVideoEnabled, setIsLiveVideoEnabled] = useState(true)
+  const [isRecoveryCaptureEnabled, setIsRecoveryCaptureEnabled] = useState(true)
+
+  useEffect(() => {
+    if (cameraData) {
+      setIsCameraEnabled(cameraData?.cameraEnabled ?? true)
```

Comment:
```suggestion
      setIsCameraEnabled(cameraData.cameraEnabled)
```
We know camera is defined given the conditional block, so we can simplify all these settings.